tests/e2e/routing.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { test, expect } from "@playwright/test";
+
+const BASE_URL = "/dengue/may24";
+const expectIndexPage = async (
+    page,
+    url: string,
+    selectedIndicator: string,
+    selectedCountry: string,
+    scale: string,
+    featureCount: number,
+    selectedCountryFeatureCount: number
+) => {
+    await page.waitForURL(new RegExp(`${BASE_URL}${url}`));
+    await expect(await page.textContent("button.bg-blue")).toBe(selectedIndicator);
+    const summary = await page.locator(".choropleth-data-summary");
+    await expect(await summary.getAttribute("selected-indicator")).toBe(selectedIndicator);
+    await expect(await summary.getAttribute("selected-country-id")).toBe(selectedCountry);
+    await expect(await summary.getAttribute("colour-scale")).toBe(scale);
+    await expect(await summary.getAttribute("feature-count")).toBe(featureCount.toString());
+    await expect(await summary.getAttribute("selected-country-feature-count")).toBe(
+        selectedCountryFeatureCount.toString()
+    );
+};
+
+test.describe("Router", () => {
+    test("browse to root redirects to base url with first indicator", async ({ page }) => {
+        await page.goto("/");
+        await expectIndexPage(page, "/FOI", "FOI", "", "interpolateReds", 33, 0);
+    });
+
+    test("browse to indicator loads expected data", async ({ page }) => {
+        await page.goto(`${BASE_URL}/p9`);
+        await expectIndexPage(page, "/p9", "p9", "", "interpolateBlues", 33, 0);
+    });
+
+    test("browse to indicator and country loads expected data", async ({ page }) => {
+        await page.goto(`${BASE_URL}/FOI/TZA`);
+        await expectIndexPage(page, "/FOI/TZA", "FOI", "TZA", "interpolateReds", 173, 170);
+    });
```

Comment:
So we don't want the functionality of 'browse to country' without specifying an indicator?

Reply:
Nope!